### PR TITLE
update Servantes with new LiveUpdate syntax

### PR DESCRIPTION
**Pull request by maiamcc**

### CODE REVIEW

**Comment by jazzdan** [comment]
I assume this replaces #85 ?

**Comment by maiamcc** [comment]
ah, yep

File: Tiltfile [hunk 1]
```diff
@@ -63,7 +61,28 @@ k8s_yaml([m4_yaml(f) for f in yamls])
 
 ## Part 2: Images
 
-# most services we do docker_builds
+docker_build('fe', 'fe',
```
Comment (jazzdan) [inline Tiltfile]: Any reason why we moved the complicated docker_builds above the simpler ones?

File: Tiltfile [hunk 2]
```diff
@@ -72,28 +91,15 @@ docker_build('words', 'words')
...
-
-# You can also ADD fast build instructions to a normal docker build; we use the fast build
-# instructions to update the live container when possible, otherwise use the docker build
-# instructions for a fresh build + deploy
-spoonerisms = docker_build('spoonerisms', 'spoonerisms')
-spoonerisms.add(repo.path('spoonerisms/src'), '/app')
-spoonerisms.add(repo.path('spoonerisms/package.json'), '/app/package.json')
-spoonerisms.add(repo.path('spoonerisms/yarn.lock'), '/app/yarn.lock')
-spoonerisms.run('cd /app && yarn install', trigger=['spoonerisms/package.json', 'spoonerisms/yarn.lock'])
+docker_build('spoonerisms', 'spoonerisms',
```
Comment (jazzdan) [inline Tiltfile]: Why is this live_update image separate from all of the others?

File: Tiltfile [hunk 3]
```diff
@@ -63,7 +61,28 @@ k8s_yaml([m4_yaml(f) for f in yamls])
 
 ## Part 2: Images
 
-# most services we do docker_builds
+docker_build('fe', 'fe',
+             live_update=[
```
Comment (landism) [inline Tiltfile]: tabs as far as the eyes can see!

we've been using 2-space tabs in this file

File: Tiltfile (hunk 2 above)
Comment (maiamcc) [reply Tiltfile]: lol no reason, i just followed the order of the original docker build declarations 😅 let me re-order

File: Tiltfile (hunk 3 above)
Comment (maiamcc) [reply Tiltfile]: 🤐

**Pull request merged**